apps/federatedfilesharing/src/components/PersonalSettings.vue

```diff
@@ -146,7 +146,7 @@ export default {
 </a>`
 		},
 		copyLinkTooltip() {
-			return this.isCopied ? t('federatedfilesharing', 'Cloud ID copied to the clipboard') : t('federatedfilesharing', 'Copy to clipboard')
+			return this.isCopied ? t('federatedfilesharing', 'Cloud ID copied to the clipboard') : t('federatedfilesharing', 'Copy')
```

Comment:
Wanna remove that section too?

```suggestion
			return this.isCopied ? t('federatedfilesharing', 'Cloud ID copied') : t('federatedfilesharing', 'Copy')
```

Reply:
Aha! Only searched for "to clipboard", not "to the clipboard".

Hold on… :)

Reply:
Surprisingly, it seems like that was the only place it appeared haha